ui/packages/docs/src/components/Frames/character_data.json

```diff
@@ -687,6 +687,14 @@
       "count": "https://docs.google.com/spreadsheets/d/1MBeFddkD4OtHdBMncqW_j18NC-lUMqA9noVo_pH26H0/edit?usp=sharing"
     }
   ],
+  "navia": [
+    {
+      "vid_credit": "shizukayuki",
+      "count_credit": "shizukayuki",
+      "vid": "https://youtu.be/qnWjXMmf6MM",
+      "count": "https://docs.google.com/spreadsheets/d/1M_FffTMTx11rbESklIbzgiSfm9RnWhgKcRlSEnJ08p4/edit#gid=0"
+    }
```

Comment:
missing 2 more entries with everything the same except the video link:
- https://youtu.be/BxIFv2HUob0
- https://youtu.be/PJu7ulF0JJo